packages/registry/src/index.ts

```diff
@@ -406,6 +420,68 @@ class Config {
     const result = currentSlotComponents.slice();
     currentSlot.data[name] = result.splice(position, 1);
   }
+
+  registerUtility(options: {
+    name: string;
+    type: string;
+    dependencies?: Record<string, string>;
+    method: (args: any) => any;
+  }) {
+    const { name, type, method, dependencies = {} } = options;
+    let depsString: string = '';
+    if (!method) {
+      throw new Error('No method provided');
+    } else {
+      depsString = Object.keys(dependencies)
+        .map((key) => `${key}:${dependencies[key]}`)
+        .join('+');
+    }
```

Comment:
I think you need to sort the keys both here and when the utility is looked up. Otherwise it will be different depending on the order in which the keys were added to the object.

Reply:
Done